docs/application/native/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,342 @@
+   The `<OwnDataPath>` refers to your own data path:
+
+   ```c
+   /* For details, see the Image Util API Reference */
+   unsigned char *dataBuffer = NULL;
+   unsigned long long bufferSize = 0;
+   unsigned long width = 0;
+   unsigned long height = 0;
+   image_util_decode_h imageDecoder = NULL;
+
+   error_code = image_util_decode_create(&imageDecoder);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_set_input_path(imageDecoder, "/<OwnDataPath>/sample.jpg");
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_set_colorspace(imageDecoder, IMAGE_UTIL_COLORSPACE_RGB888);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_set_output_buffer(imageDecoder, &dataBuffer);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_run(imageDecoder, &width, &height, &bufferSize);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   error_code = image_util_decode_destroy(imageDecoder);
+   if (error_code != IMAGE_UTIL_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   /* Fill the dataBuffer to g_source */
+   error_code = mv_source_fill_by_buffer(imagedata.g_source, dataBuffer, (unsigned int)bufferSize,
+                                         (unsigned int)width, (unsigned int)height, MEDIA_VISION_COLORSPACE_RGB888);
+   if (error_code != MEDIA_VISION_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+   free(dataBuffer);
+   dataBuffer = NULL;
+   ```
+3. To detect landmark of pose from the `sample.jpg`,
+
+    <img src="./media/mediavision_pose_sample_sumo.png" width=300>,
+    create a `g_inference` media vision inference handle:
+
+   ```c
+   error_code = mv_inference_create(&imagedata.g_inference);
+   if (error_code != MEDIA_VISION_ERROR_NONE)
+       dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+   ```
+4. Configure `g_engine_config` with a body pose model data and its mapping file. In the following example, TensorFlow Lite model is used and `data.tflite` and its mapping file `data_mapping.txt` are in `<OwnDataPath>`. Model data is available in open model zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models) or [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile). Suppose model data and its mapping file in Section <a name="background">Background</a>:
+
+    ```c
+    #define MODEL_DATA "OwnDataPath/data.tflite"
+    #define MODEL_MAPPING_FILE "OwnDataPath/data_mapping.txt"
```

Comment:
I used <> in sentences to indicate it includes a path. <OwnDataPath> is equal to OwnDataPath.
In image-classification.md, i used the same expression.
If it causes confusion, i will fix them.

Reply:
Question. It is said that in this page \<> are removed from \<OwnDataPath>.
However, in line 154 we have a OwnDataPath with \<>.
`..."/<OwnDataPath>/sample.jpg")...`

To unify the usage of \<>, I think it would be better to choose one way through the page.
We have two ways.

 - remove \<> from the `<OwnDataPath>` on line 154.
 - Apply \<> to every `OwnDataPath`s.

Indeed I prefer the second way. Would you please let me know your opinion?

Reply:
I applied <> to every `OwnDataPath`  and at line 154, i changed `"/<OwnDataPath>/sample.jpg"` to `"<OwnDataPath>/sample.jpg`